README.md

```diff
@@ -1,80 +1,67 @@
 
-These samples are built with Gradle. If you want to learn more about Gradle, learn more from [Gradle's guide](https://docs.gradle.org/current/userguide/userguide.html).
+Browse the category directories to explore the samples. Each sample is an individual [Gradle](https://docs.gradle.org/current/userguide/userguide.html) project that can be run standalone. The Gradle buildscripts have tasks for running the application, building a jar, and distributing the app as a zip.
 
-This guide has instructions for running samples using the command line or with Eclipse and IntelliJ IDEA.
+Installing Gradle is not necessary since each sample includes the Gradle wrapper.
 
-For developers new to Git, please see the wiki page for how to [clone this repository](https://github.com/Esri/arcgis-runtime-samples-java/wiki/working-with-git).
+## Instructions
 
-## Running the Samples
-To run the samples in a terminal, `cd` into the project and call the gradle `run` task, supplying the sample main class:
+### IntelliJ IDEA
 
-On Linux/Mac
-```
-$ ./gradlew run -PmainClass="com.esri.samples.scene.display_scene.DisplaySceneSample"
-```
+1. Open IntelliJ IDEA and select _File > Open..._.
+2. Choose one of the sample project directories (not the category folder) and click _OK_.
+3. Select _File > Project Structure..._ and ensure that the Project SDK and language level are set to use Java 11.
+4. Open the Gradle view with _View > Tool Windows > Gradle_.
+5. In the Gradle view, double-click the `run` task under _Tasks > application_ to run the app.
 
-On Windows
-```
-> gradlew.bat run -PmainClass="com.esri.samples.scene.display_scene.DisplaySceneSample"
-```
+### Eclipse
 
-If no `mainClass` property is supplied, the default sample (set in the build.gradle file) will be run. 
+1. Open Eclipse and select _File > Import_.
+2. In the import wizard, choose _Gradle > Existing Gradle Project_, then click _Next_.
+3. Choose one of the sample project directories (not the category folder) as the project root directory.
+4. Click _Finish_ to complete the import.
+5. Open the Gradle Tasks view with _Window > Show View > Other... > Gradle > Gradle Tasks_.
+6. In the Gradle Tasks view, double-click the `run` task under _{project_name} > application_ to run the app.
 
-There is no need to install Gradle to run the samples.
+### Terminal
 
-## Java 11
+1. `cd` into one of the sample project directories (not the category folder).
+2. Run `./gradlew run` on Linux/Mac or `gradlew.bat run` on Windows to run the app.
+
+### Java 11
 Java 11 users may find exceptions when running the project if their library path is still set for Oracle JDK 1.8 (see the [OpenJavaFX docs](https://openjfx.io/openjfx-docs/) for more information). A workaround for this is to add the following argument in the `run` task of the Gradle buildscript:
 ```
 systemProperty "java.library.path", "C:\tmp"
 ```
 
-## Offline sample data
+### Offline sample data
```

Comment:
Should this be a second-order heading `##`? Or did you intend to put it under `## Instructions`?

Reply:
Yep, these sections should all be under Instructions. Esri repos have some guidelines for the sections the readme should contain. This information fits best under their Instructions section.